src/main/resources/application.conf

```diff
@@ -6,7 +6,7 @@ centaur {
   # The maximal length of a workflow, intended as a sanity check and not actually a test in and of itself
   maxWorkflowLength: 2 hours
   # Cromwell's metadata is eventually consistent. Set a timeout such that we expect it to have eventually consisted
-  metadataConsistencyTimeout: 10 seconds
+  metadataConsistencyTimeout: 5 minutes
```

Comment:
`testCentaurLocal.sh` and `testCentaurJes.sh` (the latter delegating to `test_centaur.sh`) are primarily for Travis, perhaps that would be a narrower change?